cozy/jobs.py

```diff
@@ -17,15 +17,58 @@
    `multiprocessing` module.
 """
 
-from multiprocessing import Process, Array, Queue
+import os
+import multiprocessing
 from queue import Queue as PlainQueue, Empty, Full
 import threading
+import signal
 
 from cozy.common import partition
 from cozy.opts import Option
 
 do_profiling = Option("profile", bool, False, description="Profile Cozy itself")
 
+_interrupted = False
+def _set_interrupt_flag(signal_number, stack_frame):
+    global _interrupted
+    # print("GOT INTERRUPTED")
+    # import traceback
+    # traceback.print_stack(stack_frame)
+    _interrupted = True
+
+def handle_sigint_gracefully():
```

Comment:
nit: Do you want to name it `install_....` instead?

Reply:
Thanks, that's a good idea!